src/main/java/Lia.java

```diff
@@ -0,0 +1,316 @@
+import task.Task;
+import task.Event;
+import task.ToDo;
+import task.Deadline;
+import exception.LiaException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Lia is a simple chatbot program that helps users manage tasks.
+ */
+public class Lia {
+    private static final String INDENTATION = "    "; // Constant for Chatbot output indentation
+    private static final String FILE_PATH = "./data/lia.txt"; // Path to save tasks
+
+    public static void main(String[] args) {
+        // Customizing the chatbot with the name Lia
+        String LOGO =
+                """
+                             ██▓     ██▓ ▄▄▄     \s
+                            ▓██▒    ▓██▒▒████▄   \s
+                            ▒██░    ▒██▒▒██  ▀█▄ \s
+                            ▒██░    ░██░░██▄▄▄▄██\s
+                            ░██████▒░██░ ▓█   ▓██▒
+                            ░ ▒░▓  ░░▓   ▒▒   ▓▒█░
+                            ░ ░ ▒  ░ ▒ ░  ▒   ▒▒ ░
+                            ░ ░    ▒ ░  ░   ▒  \s
+                                ░  ░ ░        ░  ░
+                        """;
+
+        Scanner scanner = new Scanner(System.in);
+        String input;
+        ArrayList<Task> tasks = loadTasks();  // Load tasks from file
+
+        // Greet the user with enthusiasm
+        printLine();
+        System.out.println(INDENTATION + "Hello! I'm \n" + LOGO);
+        System.out.println(INDENTATION + "What can I do for you?");
+        printLine();
+
+        // Chatbot loop: keep asking for input until "bye" is entered
+        while (true) {
+            try {
+                input = scanner.nextLine().trim();
+                String[] inputArr = input.split(" ", 2);  // Splitting command and details
+
+                // If the user types "bye", end the loop with a warm farewell
+                if (inputArr[0].equalsIgnoreCase("bye")) {
+                    printLine();
+                    System.out.println(INDENTATION + "Bye. Hope to see you again soon!");
+                    printLine();
+                    break;
+                }
+
+                // Handle commands
+                handleCommand(inputArr, tasks);
+
+                // Save tasks to file after every command
+                saveTasks(tasks);
+
+            } catch (LiaException e) {
+                // Handle any Lia-specific exceptions
```

Comment:
misleading comment since you are not doing anything specific apart from printing the message.